lang/fpc/Portfile

```diff
@@ -136,6 +137,7 @@ if {${subport} eq "${name}"} {
                 -o ${destroot}${fpcbasepath}/etc/fpc.cfg
         "
         ln -s ${fpcbasepath}/etc/fpc.cfg ${destroot}${prefix}/etc
+        system "patch ${destroot}${fpcbasepath}/etc/fpc.cfg ${filespath}/fpc.cfg.patch"
```

Comment:
Does this patching have to be done manually here, or could you not use the standard macports patchfiles mechanisms instead ?

Reply:
The file to be patched (fpc.cfg) is created by fpkmcfg in the build stage. Since Patchfiles acts before the build stage, I think it needs to be this way.

Reply:
In that case this is OK, thanks.